packages/ui/aurora-theme/src/styles/fragments/surface.ts

```diff
@@ -2,13 +2,15 @@
 // Copyright 2023 DXOS.org
 //
 
-// ⚠️ NOTE: Please don’t make changes to these fragments without testing *all* the places where they’re used.
+// NOTE: Please don’t make changes to these fragments without testing *all* the places where they’re used.
 
 // If your scope of concern is narrower than exactly *the entire design system*, please define and apply your own
 // fragment, either to the specific component or in an override to `ThemeProvider`’s `tx` value.
 
+// TODO(burdon): Make able to override in theme.
+
 // Main background.
-export const baseSurface = 'bg-neutral-50 dark:bg-neutral-900';
+export const baseSurface = 'bg-neutral-12 dark:bg-neutral-900';
```

Comment:
This will affect the elevation cadence for light mode, which _might_ be fine, but e.g. this causes elements with `groupSurface` like Dialog and Popover to have a darker background than anything that would have this background.

We should check the Dialog and Popover stories, or Composer/Labs with Dialogs/Popovers open, with this token as adjusted to confirm this is what we want.

Reply:
i did check against aurora storybooks.
i would like to discuss popovers (too chatty) and dialogs (not enough contrast) in general though.